### DISCUSSION TITLE
Error introspecting an existing DB

### DISCUSSION THREAD

**Discussion by icrackl** [prisma/prisma#13358] [Q&A]
Hello!
I'm relatively new to Prisma and am trying to get introspection working on an existing DB (Microsoft SQL Server). When I run it, I am getting this error: 

```
Prisma schema loaded from prisma/schema.prisma
Datasource "db" - SQL Server

✖ Introspecting based on datasource defined in prisma/schema.prisma

Error: Error in connector: Error querying the database: Error querying the database: Error querying the database: Protocol error: header: invalid packet type: 72
```

I do not see this error if i try and run prisma against a new DB with some dummy data. Any ideas on where to start with debugging this? I haven't been able to find any references to that sort of error message online. 

The schema file is relatively simple:
```
datasource db {
  provider = "sqlserver"
  url      = "sqlserver://localhost:4000;database=rad;user=SA;password=SuperSecret100;encrypt=true;"
}
```

**Comment by nurul3101** [comment]
Hey @icrackl 👋 
What's your prisma version? Is your repository public or could you provide us with a sample reproduction?

**Reply by vladi-stevanovic** [reply]
👋 Hello @icrackl  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
